stores/healthcareProfessionalsStore.ts

```diff
@@ -15,9 +15,22 @@ export const useHealthcareProfessionalsStore = defineStore(
             healthcareProfessionalsData.value = healthcareProfessionalResults
         }
 
+        async function updateHealthcareProfessional(healthcareProfession: MutationUpdateHealthcareProfessionalArgs) {
+            try {
+                return await graphQLClientRequestWithRetry(
+                    gqlClient.request.bind(gqlClient),
+                    updateHealthcareProfessionalGqlMutation,
+                    healthcareProfession
```

Comment:
```suggestion
                    healthcareProfessional
```

Reply:
whoops! Thanks for the catch. Definitely meant to do professional